angular-dragula.js

```diff
@@ -2,36 +2,15 @@
 
 var dragula = require('dragula');
 var dragulaService = require('./service');
-var replicateEvents = require('./replicate-events');
+var dragulaDirective = require('./directive');
 
 function register (angular) {
   var app = angular.module('dragula', ['ng']);
 
-  app.factory('dragulaService', dragulaService);
-  app.directive('dragula', ['dragulaService', angularDragula]);
+  app.factory('dragulaService', dragulaService(angular));
+  app.directive('dragula', ['dragulaService', dragulaDirective(angular)]);
```

Comment:
Please move the entire array to the directive, as there's no reason why this module should know the dependencies of the other one, nor why we would want to have to change the dependencies in two files.